File: src/easy/FindContentChildren.swift
func findContentChildren(_ children: [Int], _ cookies: [Int]) -> Int {
  if children.isEmpty || cookies.isEmpty {
    return 0
  }
  let sortedChildren = children.sorted()
  let sortedCookies = cookies.sorted()

  var i = 0
  var j = 0
  while i < sortedChildren.count && j < sortedCookies.count {
    if sortedCookies[j] >= sortedChildren[i] {
      i += 1
    }
    j += 1
  }
  return i
}

File: lib/easy/find_content_children.dart
int findContentChildren(List<int> children, List<int> cookies) {
  if (children.isEmpty || cookies.isEmpty) return 0;
  int count = 0;
  int i = 0;
  int j = 0;
  children.sort();
  cookies.sort();
  while (i < children.length && j < cookies.length) {
    if (cookies[j] >= children[i]) {
      count += 1;
      i += 1;
    }
    j += 1;
  }
  return count;
}
